integration/spark-common/src/main/scala/org/apache/spark/util/FileUtils.scala

```diff
@@ -73,7 +73,7 @@ object FileUtils {
       throw new DataLoadingException("Input file path cannot be empty.")
     } else {
       val stringBuild = new StringBuilder()
-      val filePaths = inputPath.split(",")
+      val filePaths = inputPath.split(",").map(_.trim)
```

Comment:
Can you please add a testcase for this scenario

Reply:
done